app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -142,6 +142,23 @@ def billed?
           !!doc_auth_result
         end
 
+        def doc_auth_success?
+          transaction_status_passed? &&
+            true_id_product.present? &&
+            product_status_passed? &&
+            doc_auth_result_passed?
+        end
+
+        # @return [Boolean, nil]
+        # When selfie check is not required, return nil
+        # Otherwise:
+        #   return true if selfie check result == 'Pass'
+        #   return false
+        def selfie_success
```

Comment:
can we make this method private?

Reply:
@amirbey , it is intentionally to be public, my reasoning is that it's the interface where various Response implementations to be provided. It's contract provided to clients of the Response class.